### PR TITLE
Conda release pipeline

### PR DESCRIPTION
**Pull request by bschroeter**
Added a conda release pipeline using the non-vendored mode of versioneer, which avoids the addition of a versioneer.py file in the root of the repository.

This approach correctly generates version numbers in all expected locations and permits publishing direct to conda without PyPI (though we could easily add that).

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          environment-file: .conda/build_env.yaml
+          auto-update-conda: false
+          auto-activate-base: false
+          show-channel-urls: true
+      - name: Build and upload the conda packages
+        uses: uibcdf/action-build-and-upload-conda-packages@v1.2.0
+        with:
+          python-version: '3.9'
+          meta_yaml_dir: .conda
+          user: bschroeter
```
Comment (ccarouge) [inline .github/workflows/cd.yml:26]: wrong user.

**Review by ccarouge — COMMENTED** [review]
I'm not putting a whole review but I think the user needs to be changed in the CD workflow.

**Comment by bschroeter** [comment]
@ccarouge, yes of course. I am aware of this and it will be changed in a subsequent issue once this process is verified on the main repo using my profile. I didn’t want to make a mess of the official channel until this was nailed down.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: .github/workflows/cd.yml (hunk 1 above)
Comment (bschroeter) [reply .github/workflows/cd.yml:26]: Referenced in https://github.com/CABLE-LSM/meorg_client/issues/38 to remind myself to do this once this PR is closed.

File: .github/workflows/cd.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          label: main
+          token: ${{ secrets.ANACONDA_TOKEN_BEN }}
```
Comment (bschroeter) [inline .github/workflows/cd.yml:28]: Same here.

**Review by Whyborn — APPROVED** [review]

File: .github/workflows/cd.yml (hunk 1 above)
Comment (ccarouge) [reply .github/workflows/cd.yml:26]: Personally, I would have found it useful to be stated in the PR description. Which I conscientiously read before reviewing code.

Comment (bschroeter) [reply .github/workflows/cd.yml:26]: Noted. This is a good suggestion all around to put more detail in our PR descriptions. Thanks.

**Pull request merged**